data/p5js/mountain-view/index.md

```diff
@@ -1,11 +1,15 @@
 ---
 layout: p5js
-authors: [Edmundo Mejia Galindo, Julia Hilt]
-authors-url: [https://github.com/edgalindo,https://github.com/piixelcat]
-title: [Mountain view: Berchtesgaden – Nussdorf]
-tags: [P5.js,[data], [color], [form], [repeat], Law of Something]
-permalink: /[data]/[Mountain view: Berchtesgaden – Nussdorf]/
-summary: [A simple data visualization based on altitude profiles forms abstract mountain views.]
+authors:
+  - "Edmundo Mejia Galindo"
+  - "Julia Hilt"  
+authors-url:
+ - "https://github.com/edgalindo"
+ - "https://github.com/piixelcat" 
+title: Mountain view: Berchtesgaden – Nussdorf
+tags: [P5.js, data, color, form, repeat, Law of Continuity]
+permalink: /data/Mountain view: Berchtesgaden – Nussdorf/
```

Comment:
To last changes. The permalink needs to be URL save. No whitespace, no : no uppercase. hyphens are okay.  

Please add for now also the:  

```
author: Julia Hilt
author-url:  "https://github.com/piixelcat" 
```

so it will be parsed correctly. I'll need some time to update the site to allow multiple authors.